src/select.coffee

```diff
@@ -400,6 +400,9 @@ Ember.AddeparMixins.ResizeHandlerMixin, Ember.Widgets.KeyboardHelper,
     )
   .property 'groupedContent.[]'
 
+  # whether or not to ensure that highlighted option is visible
```

Comment:
This variable controls whether the function `@ensureVisiable` is called or not, while the function `@ensureVisible` controls how to ensure visibility, it can be scrolling or other way. So the name is aligned IMO.